Dexterous-Rambutan/battle-code
# Merge with hahnbi

#### What's this PR do?

Successfully connected both redisQueues from server to worker and back to server, after testing the solution string. Is now able to handle errors.
#### What are the important parts of the code?

workers/ responseRedis/ and solutionController
#### How should this be tested by the reviewer?
1. run npm install
2. Start Postgres
3. run redis-server
4. run node server/server.js
5. run node worker/solutionTester.js
6. Nave to http://127.0.0.1:3000/api/resetDB
7. Sign in to github as kweng2
8. Nav to http://127.0.0.1:3000/addProblemsSolutions.html
9. Add nthFibonacci challenge
10. Write nthFibonacci solution and click submit, with challenge id field as 1
11. Expect server to push test to testQueue, and solutionTester to pop test from testQueue
12. Expect solutionTester to test the solution code and console log errors or succes
13. Expect solutionTester to push to rQueue, and server to pop result from rQueue
#### Is any other information necessary to understand this?
#### What are the relevant tickets? (Please add `closes`, `refs`, etc)

Closes #30 
Closes #28
Refs #16 
#### Screenshots (if appropriate)
